src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_InvalidExpression.cs

```diff
@@ -0,0 +1,390 @@
+
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.CSharp.Test.Utilities;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Roslyn.Test.Utilities;
+using System.Linq;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests
+{
+    public partial class IOperationTests : SemanticModelTestBase
+    {
+        [Fact, WorkItem(17598, "https://github.com/dotnet/roslyn/issues/17598")]
+        public void InvalidInvocationExpression_BadReceiver()
+        {
+            string source = @"
+using System;
+
+class Program
+{
+    static void Main(string[] args)
+    {
+        /*<bind>*/Console.WriteLine2()/*</bind>*/;
+    }
+}
+";
+            string expectedOperationTree = @"
+IInvocationExpression ( ? ?.()) (OperationKind.InvocationExpression, Type: ?, IsInvalid)
+  Instance Receiver: IInvalidExpression (OperationKind.InvalidExpression, Type: ?, IsInvalid)
+      IOperation:  (OperationKind.None)
+";
+            VerifyOperationTreeForTest<InvocationExpressionSyntax>(source, expectedOperationTree);
+        }
+
+        [Fact, WorkItem(17598, "https://github.com/dotnet/roslyn/issues/17598")]
+        public void InvalidInvocationExpression_OverloadResolutionFailureBadArgument()
+        {
+            string source = @"
+using System;
+
+class Program
+{
+    static void Main(string[] args)
+    {
+        /*<bind>*/F(string.Empty)/*</bind>*/;
+    }
+
+    void F(int x)
+    {
+    }
+}
+";
+            string expectedOperationTree = @"
+IInvocationExpression ( void Program.F(System.Int32 x)) (OperationKind.InvocationExpression, Type: System.Void, IsInvalid)
+  Instance Receiver: IInstanceReferenceExpression (InstanceReferenceKind.Implicit) (OperationKind.InstanceReferenceExpression, Type: Program)
+  IArgument (Matching Parameter: x) (OperationKind.Argument)
+    IFieldReferenceExpression: System.String System.String.Empty (Static) (OperationKind.FieldReferenceExpression, Type: System.String)
+";
+            VerifyOperationTreeForTest<InvocationExpressionSyntax>(source, expectedOperationTree);
+        }
+
+        [Fact(Skip = "https://github.com/dotnet/roslyn/issues/8813"), WorkItem(17598, "https://github.com/dotnet/roslyn/issues/17598")]
+        public void InvalidInvocationExpression_OverloadResolutionFailureExtraArgument()
+        {
+            string source = @"
+using System;
+
+class Program
+{
+    static void Main(string[] args)
+    {
+        /*<bind>*/F(string.Empty)/*</bind>*/;
+    }
+
+    void F()
+    {
+    }
+}
+";
+            string expectedOperationTree = @"
+IInvocationExpression ( void Program.F(System.Int32 x)) (OperationKind.InvocationExpression, Type: System.Void, IsInvalid)
+  Instance Receiver: IInstanceReferenceExpression (InstanceReferenceKind.Implicit) (OperationKind.InstanceReferenceExpression, Type: Program)
+  IArgument (Matching Parameter: x) (OperationKind.Argument)
```

Comment:
Ah, because the next test is skipped, this wasn't caught. Let me fix the next test.